### ISSUE TITLE
Video Not create

### ISSUE THREAD

**Issue by hassanalimughal007** [Hammad2111/VideoEditor-GlaxyAppszone#1]
In watermark feature i save the video but all times the same error are occur
**Command failed with state COMPLETED and rc 1.null**
why this error are occurs all the time, Although i give the proper storage address

**Comment by Hammad2111** [comment]
yes there are some issues i am working to sort it out soon it will be updated

**Comment by hassanalimughal007** [comment]
Another issue rise
if i merge the 2 music videos, it merged but it play the first video background music not second, the second merge video is mute
